data/json/items/appliances.json

```
@@ -270,7 +270,7 @@
     "longest_side": "170 cm",
     "bashing": 5,
     "price": 90000,
-    "price_postapoc": 500,
+    "price_postapoc": 5000,
```

```suggestion
    "price_postapoc": 1500,
```
three times the value rather than ten